frontend/src/components/projectDetail/tests/shareButton.test.js

```diff
@@ -31,6 +31,7 @@ describe('test if shareButton', () => {
 
   it('render open corresponding window popup', async () => {
     global.open = jest.fn();
+    const user = userEvent.setup();
```

Comment:
It seems like the you've forgotten to replace `userEvent` with the declared `user` variable in the test code